README.md

```diff
@@ -8,51 +8,22 @@
 [![Swagger](https://img.shields.io/badge/-Swagger-%23Clojure?style=for-the-badge&logo=swagger&logoColor=white)](https://funding-service-design-account-store-dev.london.cloudapps.digital/#/default/core.account.post_account_by_email)
 ![Gunicorn](https://img.shields.io/badge/gunicorn-%298729.svg?style=for-the-badge&logo=gunicorn&logoColor=white)
 
-Repo for the funding service design account store.
+This repository is designed to handle the generating and providing of JSON payloads for accounts. It takes an email address and creates/returns a corresponding payload of account json
```

Comment:
Second sentence doesn't really make sense as an overview of the repo?

Reply:
have shortened to just first sentence overview